docs/content/dagster-cloud/managing-deployments/dagster-cloud-yaml.mdx

```diff
@@ -0,0 +1,298 @@
+At a minimum, each item in `locations` must include the following: 
+- A `location_name` key that specifies the name of the code location
+- A `code_source` key containing either a `module_name` or `file_name` parameter that specifies where to find the definitions in the code location
+Refer to the following table for the full list of parameters a location can have.
+
+Let's put these examples into tabs:
+
+<TabGroup>
+<TabItem name="Single code location">
+
+```yaml
+# dagster_cloud.yaml
+
+locations:
+  - location_name: data-eng-pipeline
+    code_source:
+      package_name: quickstart_etl
+```
+</TabItem>
+<TabItem name="Multiple code locations">
+
+```yaml 
+#dagster_cloud.yaml
+
+locations:
+  - location_name: data-eng-pipeline
+    code_source:
+      package_name: quickstart_etl
+  - location_name: machine_learning
+    code_source:
+      python_file: ml/ml_model.py
+```
+</TabItem>
+</TabGroup>
+
+### Working directory
+
+There might be some cases where you want to load your Dagster code from a different directory than the root of your code repository. You can specify the directory you want to load your Dagster code from in the dagster_cloud.yaml file using the `working_directory` option.
+
+
+Example file directory:
+
+```shell
+quickstart_etl
+├── README.md
+├── project_directory
+│   ├── quickstart_etl
+│     ├── __init__.py
+│     ├── assets
+│   ├── quickstart_etl_tests
+├── dagster_cloud.yaml
+├── pyproject.toml
+├── setup.cfg
+└── setup.py
+```
+Example:
+
+```yaml
+#dagster_cloud.yaml
+
+locations:
+  - location_name: data-eng-pipeline
+    code_source:
+      package_name: quickstart_etl
+    working_directory: ./project_directory
+```
+
+### Build directory
+
+Setting a build directory is useful if your `setup.py` or `requirements.txt` is in a subdirectory instead of the project root. This is common if you have multiple Python packages within a single Dagster project.
+
+The build registry is applicable to hybrid deployments only.
+
+Example:
+
+```yaml
+# dagster_cloud.yaml
+
+locations:
+  - location_name: data-eng-pipeline
+    code_source:
+      package_name: quickstart_etl
+    build:
+      directory: ./
+      registry: your-docker-image-registry/image-name
+```
+
+### Python executable
+
+If you are using Dagster Cloud Serverless, the Python executable to use is specified [here](https://docs.dagster.io/dagster-cloud/deployment/serverless#using-a-different-python-version). For Dagster Cloud Hybrid, the python executable that is installed globally in the image, or the default Python executable on the local system if you use the local agent, will be used. If you want to use a different python executable, you can specify it in the dagster_cloud.yaml file. It might also be useful to have different python executables for different code locations.
```

Comment:
My comment on the old PR is still relevant IMO: https://github.com/dagster-io/dagster/pull/13646/files#r1172942811

Reply:
Serverless wouldnt be relevant here though -> should i link to the UI stuff?

Reply:
I'm confused about this bit then - what is a Serverless user supposed to do with this setting?